addons/mail/static/src/components/thread_needaction_preview/thread_needaction_preview.js

```diff
@@ -73,8 +73,6 @@ class ThreadNeedactionPreview extends Component {
     }
 
     /**
-     * Get inline content of the last message of this conversation.
```

Comment:
don't remove existing comment

Reply:
:facepalm:  sorry

Reply:
There's still diff on the file? Added space?